tests/backup/backup_restore_basic_test.go

```diff
@@ -1073,28 +1114,29 @@ var _ = Describe("{AllNSBackupWithIncludeNewNSOption}", func() {
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Inspecting next schedule backup named [%s] and uid [%s]", nextScheduleBackupName, nextScheduleBackupUid))
 			backedUpNamespaces := resp.GetBackup().GetNamespaces()
 			log.InfoD("Namespaces in next schedule backup named [%s] and uid [%s] are [%v]", nextScheduleBackupName, nextScheduleBackupUid, backedUpNamespaces)
-			for _, namespace := range appNamespaces {
+			for _, namespace := range newAppNamespaces {
 				dash.VerifyFatal(strings.Contains(strings.Join(backedUpNamespaces, ","), namespace), true, fmt.Sprintf("Checking the new application namespace [%s] against the next scheduled backup named [%s]", namespace, nextScheduleBackupName))
 			}
 		})
-		Step("Restore new application namespaces from next schedule backup", func() {
-			log.InfoD("Restoring new application namespaces from next schedule backup")
+		Step("Restore new application namespaces from next schedule backup in source cluster", func() {
+			log.InfoD("Restoring new application namespaces from next schedule backup in source cluster")
 			ctx, err := backup.GetAdminCtxFromSecret()
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			namespaceMapping := make(map[string]string)
 			// Modifying namespaceMapping to restore only new namespaces
-			for _, namespace := range appNamespaces {
-				namespaceMapping[namespace] = namespace
+			for _, namespace := range newAppNamespaces {
+				namespaceMapping[namespace] = "restored-" + namespace
```

Comment:
Here we would be restoring only the newly created namespace and not the ones created earlier.

Also add `restored` string as a suffix instead of prefix

Reply:
I am assuming, if we create a new namespaceMapping that solely comprises newAppNamespaces, it will only restore newAppNamespaces. I have added "restored" as suffix.